hedyorg/hedy
# 🪲 fix programs page

The program overview currently throws an error. 
![image](https://github.com/hedyorg/hedy/assets/20051470/c56cb40e-4ff0-4e1a-9518-c9d6140ff6c4)

This PR fixes it and makes sure that we show a custom adventure's name and not id in the program block.

**How to test?**
- run a program/adventure 
- go to my programs
- it should work. 
- stop the server
- remove the `adventure_name` field from that program/adventure's in dev_database > programs > your program
- re-run the server
- my programs page should still run fine!
